core-java/src/test/java/com/baeldung/system/SystemPropertiesTest.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.system;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Properties;
+
+public class SystemPropertiesTest {
+
+    @Test
+    public void givenSystem_whenCalledGetProperties_thenReturnPropertiesinResult() {
```

Comment:
Let's separate these out into separate tests. (The tests look great, by the way!) It will be easier for your readers to process if you are just testing one thing at a time.